Mail2Bug/Email/EWS/EWSConnectionManger.cs

```diff
@@ -58,7 +58,7 @@ public EWSConnection GetConnection(Credentials credentials)
                 }
 
                 Logger.InfoFormat("Creating FolderMailboxManager for {0}", key);
-                _cachedConnections[key] = ConnectToEWS(credentials);
+                _cachedConnections[key] = ConnectToEWS(credentials, useConversationGuidOnly);
```

Comment:
Ideally, I'd say the distinction should be made in the processing strategy, but if not there, then at the very least it should not be in the core (cache-able) connection obejct, but rather in an object that wraps it.